index.js

```diff
@@ -160,7 +191,26 @@ function activateSortBtns() {
   sortBtns.forEach((btn) => btn.addEventListener("click", handleSortBtnClick));
 
   function handleSortBtnClick(e) {
-    console.log(e.target.value);
+    // testing block
+    console.log("flags BEFORE click");
+    console.log("sortingBy ", sortingBy);
+    console.log("sortingOrder", sortingOrder);
+    // end of testing block
+    if (sortingBy !== e.target.value) {
```

Comment:
We should change the buttons' value attributes to match the `topic` parameters they are being sorted by.